src/track.cpp

```diff
@@ -128,20 +135,19 @@ void Track::incoming(message_ptr message) {
 }
 
 void Track::setRtcpHandler(std::shared_ptr<RtcpHandler> handler) {
-	if (mRtcpHandler)
-		mRtcpHandler->onOutgoing(nullptr);
-
 	mRtcpHandler = std::move(handler);
-	if (mRtcpHandler) {
-		mRtcpHandler->onOutgoing([&]([[maybe_unused]] message_ptr message) {
-#if RTC_ENABLE_MEDIA
-			if (auto transport = mDtlsSrtpTransport.lock())
-				transport->sendMedia(message);
-#else
-			PLOG_WARNING << "Ignoring RTCP send (not compiled with SRTP support)";
-#endif
-		});
-	}
+	mRtcpHandler->onOutgoing([&](const rtc::message_ptr& message) {
```

Comment:
You'll need a `[[maybe_unused]]` attribute on the `message` parameter to prevent an unused variable if media support is disabled.